### ISSUE TITLE
Nan values using PPO training methods

### ISSUE THREAD

**Issue by TiagomcMoreira** [ScheiklP/sofa_zoo#13]
Hey, I'm quite new with coding and SB3 so this might be a simple thing, but while I'm using PPO to train my custom environment in which my learning rate is dependant on the progress inside the model PPO ( def lr(progress):(...) and then model = PPO(learning_rate = lr) ) and some Nan values ocurred
After that I introduced some prints inside the function to check on the progress variable and noticed it went to negative values whereas it should be between 0 and 1. Anyone knows anything about this?

**Comment by ScheiklP** [comment]
Hi @TiagomcMoreira ,
since this question is a more general topic, and does not relate to sofa_zoo, could you post it over at https://github.com/DLR-RM/stable-baselines3?

Cheers,
Paul
